tensorflow/lite/arena_planner.cc

```diff
@@ -197,7 +197,7 @@ TfLiteStatus ArenaPlanner::ExecuteAllocations(int first_node, int last_node) {
   dealloc_node_.resize(graph_info_->num_tensors(), kNodeNotAssigned);
   allocs_.resize(graph_info_->num_tensors());
   // Set allocation and deallocation for temporary tensors.
-  for (size_t i = first_node; i <= last_node && i < graph_info_->num_nodes();
+  for (size_t i = first_node; i <= (size_t)last_node && i < (size_t)(graph_info_->num_nodes());
```

Comment:
graph_info_->num_nodes() returns size_t already.

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/graph_info.h#L38

Reply:
Yes agreed. The warning is resolved by only casting last_node to size_t.